pandas/tests/io/test_common.py

```diff
@@ -513,19 +513,18 @@ def test_is_fsspec_url_chained():
     assert not icom.is_fsspec_url("filecache::://pandas/test.csv")
 
 
-@pytest.mark.parametrize("encoding", [None, "utf-8"])
```

Comment:
I could also keep testing this but CPython warns about using `open(..., encoding=None)`: `EncodingWarning: 'encoding' argument not specified`. In this case we *are* explicitly specifying `None` but I don't think CPython can tell the difference.